### PR TITLE
swap to a python based config file and add a deleted message tracker

### PR DESCRIPTION
**Pull request by xela-zone**

### CODE REVIEW

**Comment by nullpixel** [comment]
i love yaml tho

**Comment by xela-zone** [comment]
how bout now bby

i actually got a feature working

File: cogs/events.py [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
         self.bot = bot
+    
+    async def on_message(self, message):
+        print(message.server.id)
+    
+    async def on_message_edit(self, before, after):
+        pass
+    
+    async def on_message_delete(self, message):
+        log_channel = self.bot.get_channel(config.guilds[message.server.id]['server-log'])
```
Comment (nullpixel) [inline cogs/events.py]: * You could make this global to the class. 
* Why doesn't the bot automatically find and make one if it does not exist? Could that be a command?

File: cogs/events.py [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+        log_message = ":wastebasket: " + message.author.name + "#" + message.author.discriminator + "`" + message.author.id + "` deleted from channel #" + message.channel.name + "\n" + message.content
```
Comment (nullpixel) [inline cogs/events.py]: Could use .format() here for a cleaner look?

**Review by nullpixel — CHANGES_REQUESTED** [review]

**Comment by nullpixel** [comment]
Okay, merging. When you have time, I'd love a !setup command to automatically make all required channels and add them to the config.

Still hurt you took away my yaml 😦

**Review by nullpixel — APPROVED** [review]

**Pull request merged**